scout/constants/indexes.py

```diff
@@ -100,15 +100,16 @@
                 ("case_id", ASCENDING),
                 ("variant_type", ASCENDING),
                 ("rank_score", ASCENDING),
+                ("hgnc_ids", ASCENDING),
             ],
-            name="category_caseid_varianttype_rankscore",
+            name="category_caseid_varianttype_rankscore-hgncids",
         ),
         IndexModel(
             [
+                ("chromosome", ASCENDING),
```

Comment:
This is also the same as the first, just making it a bit easier on the scheduler.